### DISCUSSION TITLE
Is this error of SpatialCrop?

### DISCUSSION THREAD

**Discussion by danny4159** [Project-MONAI/MONAI#4516] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/52225997/174025191-70b22c08-bcc9-475e-aa47-84f664fdf6ef.png)

Hello. I am a student studying medical image processing.

I used SpatialCrop to crop 3d patches of brain lesions.
I set the center point of the lesion as roi_center and call the size of the lesion as roi_size, but a problem occurred.
The expected image size is (10,10,10), but the result is (180,10,10).

If I adjust roi_size, the second and third parameters change, but the first parameter is fixed as it is the size of input_data.

Thank you for your attention.

**Comment by Nic-Ma** [comment, marked as answer]
Hi @danny4159 ,

Thanks for your interest here.
Please note that most of MONAI transforms expect `channel-first` input image.
So I think here you need to add `AddChannel` transform after `LoadImage`.

Thanks.
